service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/EpisodeOfCareMapperTest.java

```diff
@@ -116,7 +116,7 @@ public void mapEpisodeOfCareNoResponsibleParty() {
 
             if (assignedEntity.isSetRepresentedOrganization()) {
                 POCDMT000002UK01Organization representedOrganization = assignedEntity
-                    .getRepresentedOrganization();
+                        .getRepresentedOrganization();
```

Comment:
please remove additional spacing here

Reply:
This is the code formatter, not added intentionally